lib/MusicBrainz/Server/Controller/User.pm

```diff
@@ -440,7 +440,7 @@ sub tag : Chained('load') PathPart('tag') Args(1)
     my $tag = $c->model('Tag')->get_by_name($tag_name);
     my %tags = ();
     my $tag_in_use = 0;
-    my @entities_with_tags = sort { $a <=> $b } entities_with('tags');
+    my @entities_with_tags = sort { $a cmp $b } entities_with('tags');
```

Comment:
Hi Ulrich,
I don’t know what `$a` and `$b` are in `$a cmp $b` but is this sorting the same order, whatever the language is ? :)